packages/vuetify/src/locale/en.ts

```diff
@@ -44,4 +44,13 @@ export default {
     am: 'AM',
     pm: 'PM',
   },
+  pagination: {
+    ariaLabel: {
+      wrapper: 'Pagination Navigation',
+      next: 'Next page',
+      previous: 'Previous page',
+      page: 'Goto Page {0}',
+      currentPage: 'Current Page, Page {0}',
```

Comment:
```suggestion
      currentPage: 'Current page: {0}',
```
This sounds better for me, applies for other languages too